GeometryService/inc/PSEnclosureMaker.hh

```diff
@@ -24,8 +24,10 @@ namespace mu2e {
 
                                             // The center of the downstream surface of the PS
                                             const CLHEP::Hep3Vector& psEndRefPoint);
-  };
+  private:
+    static void readWindow(int i, const SimpleConfig& c, const CLHEP::Hep3Vector& winRefPoint, std::unique_ptr<PSEnclosure>& res);
```

Comment:
I just noticed this now.  There is no reason for this to be  static member function.  If you can make it a free function in an anonymous namespace in the same file, that is the preferred solution.  A regular member function is also OK.

Reply:
In the interest of moving this forward, please make it just a regular member function and add a comment to suggest a future improvement.  Then I think we are ready to merge.

Reply:
This was originally an anonymous namespace free function, as you suggested, but then I wanted to access/alter private data members of PSEnclosure and this didn't work when I first tried. If I try to make just this function non-static, it fails to compile since the `make` function is static and calls this function. Would you suggest I add public functions that alter the private data of the PSEnclosure object to return this function to being an anonymous namespace free function?

Reply:
You are right that if it modifies private member data of the PSEnclosure object then a free function won't work.  You are right that a static member function may not call a non-static one.  The problem is that the function make should not have been static. Let's leave it as it is since I don't know where the tentacles lead and there are many similar instances to clean up.   

So I think we are done with this review.  Thanks for your work on this and for your patience.

Reply:
One last question.  The comment on the PR says that you updated geom_common_current.txt but I don't see it among the modified files.  Which did you intend?

Reply:
Strange! I had changed this file, but I must have accidentally reverted it at some point before committing it. I'll update this again, thank you for catching this!